buildspecs/release-to-maven-central.yml

```diff
@@ -0,0 +1,57 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - pip install awscli --upgrade --user
+
+  pre_build:
+    commands:
+      - ROOT=`pwd`
+      - SETTINGS_XML_TEMPLATE=buildspecs/resources/maven-central-release-settings.xml
+      - SETTINGS_XML=release-settings-final.xml
+      - SDK_SIGNING_GPG_SECRING=secring.gpg
+      - SDK_SIGNING_GPG_SECRING_ARN="arn:aws:secretsmanager:us-east-1:103431983078:secret:sdk-signing-gpg-secret-ring-9d0YXc"
+      - SDK_SIGNING_GPG_KEYNAME_ARN="arn:aws:secretsmanager:us-east-1:103431983078:secret:sdk-signing-gpg-keyname-wFsOOg"
+      - SDK_SIGNING_GPG_PASSPHRASE_ARN="arn:aws:secretsmanager:us-east-1:103431983078:secret:sdk-signing-gpg-passphrase-A0H1Kq"
+      - SONATYPE_PASSWORD_ARN="arn:aws:secretsmanager:us-east-1:103431983078:secret:maven-central-publishing-password-yktnUc"
+      - SONATYPE_USERNAME_ARN="arn:aws:secretsmanager:us-east-1:103431983078:secret:maven-central-publishing-username-RDvOnW"
+      - MODULES_TO_SKIP="protocol-tests,protocol-tests-core,codegen-generated-classes-test,sdk-benchmarks,module-path-tests,tests-coverage-reporting,stability-tests,sdk-native-image-test,auth-tests,s3-benchmarks,http-client-benchmarks,region-testing,old-client-version-compatibility-test,crt-unavailable-tests,bundle-shading-tests,v2-migration-tests,architecture-tests,s3-tests"
+
+  build:
+    commands:
+      - RELEASE_VERSION=`mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec`
+      - ARTIFACT_URL="https://repo1.maven.org/maven2/software/amazon/awssdk/aws-sdk-java/$RELEASE_VERSION/"
```

Comment:
Just curious about this change. With OSSRH, we [query it directly](https://github.com/aws/aws-sdk-java-v2/blob/5dac4897805c5dc102e149d32eab4bb90375bff6/buildspecs/release-to-maven.yml#L24) since there's a lag between publishing to the nexus instance and it being available through Maven (i.e. at `https://repo1.maven.org`).

Is there a similar "pre-Maven" URL that we can hit for Central publishing?

Reply:
This is actually a great question. I changed this to target the CDN because the existing polling endpoint "https://aws.oss.sonatype.org/service/local/repositories/releases/content/software/amazon/awssdk/aws-sdk-java/$RELEASE_VERSION/" is hosted on the nexus2 server. I was not sure if this will "go away" and if central will have it own server for hosting. I assume if it does happen the CDN will just get routed to that.

Reply:
Do you think we should keep it pointing to the oss endpoint?

Reply:
> Do you think we should keep it pointing to the oss endpoint?

Does that work though? I'm assuming since this is a completely different publishing mechanism, that Nexus isn't involved at all so can't really do that. Can we ask Sonatype?

Reply:
On the other hand, if publishing to Central portal makes it show up immediately in the CDN, that works too.